lib/git_reflow/git_server/pull_request.rb

```diff
@@ -3,6 +3,24 @@ module GitServer
     class PullRequest
       attr_accessor :description, :html_url, :feature_branch_name, :base_branch_name, :build_status, :source_object, :number
 
+      APPROVAL_REGEX = /(?i-mx:lgtm|looks good to me|[:\\+1:]|:thumbsup:|:shipit:)/
```

Comment:
Ah yes, I modified the `:+1:` expression in the regex. Rubular is telling me that it matches, but it seems to split it into different tokens. You are right, let's use `(?i-mx:lgtm|looks good to me|:\+1:|:thumbsup:|:shipit:)` instead! Good catch!